test/conftest.py

```diff
@@ -1,12 +1,14 @@
 import pytest
 
 skip_openai = False
+skip_redis = False
 
 
-# Registers command-line option '--skip-openai' via pytest hook.
+# Registers command-line option '--skip-openai' and '--skip-redis' via pytest hook.
 # When this flag is set, it indicates that tests requiring OpenAI should be skipped.
```

Comment:
Second part of the comment is inconsistent, better be updated:
`When any of the flags is set, tests requiring OpenAI or Redis (correspondingly) will be skipped`